### PR TITLE
Minor stylistic adjustments to UpsertEntityTagsAtomicOperation and Spec

### PR DESCRIPTION
**Pull request by ajordens**

### CODE REVIEW

**Comment by ajordens** [comment]
PTAL @anotherchrisberry ... super minor things

File: clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/ops/UpsertEntityTagsAtomicOperation.java [hunk 1]
```diff
@@ -55,9 +56,14 @@ public UpsertEntityTagsAtomicOperation(Front50Service front50Service,
...
   public Void operate(List priorOutputs) {
     if (entityTagsDescription.getId() == null) {
-      setIdAndIdPattern(entityTagsDescription);
+      EntityRefIdBuilder.EntityRefId entityRefId = entityRefId(accountCredentialsProvider, entityTagsDescription);
+
+      entityTagsDescription.setId(entityRefId.id);
+      entityTagsDescription.setIdPattern(entityRefId.idPattern);
     }
 
+    Date now = new Date();
```
Comment (ajordens) [inline clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/ops/UpsertEntityTagsAtomicOperation.java]: Passing a `Date` around lets us write a slightly better test case

File: clouddriver-elasticsearch/src/test/groovy/com/netflix/spinnaker/clouddriver/elasticsearch/ops/UpsertEntityTagsAtomicOperationSpec.groovy [hunk 2]
```diff
@@ -92,18 +94,24 @@ class UpsertEntityTagsAtomicOperationSpec extends Specification {
...
+
     when:
-    operation.mergeExistingTagsAndMetadata(current, description)
+    operation.mergeExistingTagsAndMetadata(now, current, description)
 
     then:
     description.tagsMetadata.tag1.created == 1L
     description.tagsMetadata.tag1.createdBy == "chris"
-    description.tagsMetadata.tag1.lastModified != 2L
+    description.tagsMetadata.tag1.lastModified == now.time
```
Comment (ajordens) [inline clouddriver-elasticsearch/src/test/groovy/com/netflix/spinnaker/clouddriver/elasticsearch/ops/UpsertEntityTagsAtomicOperationSpec.groovy]: The slightly better test case bit

Comment (anotherchrisberry) [inline clouddriver-elasticsearch/src/test/groovy/com/netflix/spinnaker/clouddriver/elasticsearch/ops/UpsertEntityTagsAtomicOperationSpec.groovy]: 👍

**Comment by anotherchrisberry** [comment]
LGTM, thanks for these changes

**Pull request merged**